srcts/src/components/_utils.ts

```diff
@@ -50,10 +50,26 @@ function doWindowResizeOnElementResize(el: HTMLElement): void {
   $(el).data("window-resize-observer", ro);
 }
 
+function getAllFocusableChildren(el: HTMLElement): HTMLElement[] {
+  const selectors = [
+    "[href]",
+    "input:not([disabled])",
+    "button:not([disabled])",
+    "select:not([disabled])",
+    "summary:not(:disabled)",
```

Comment:
Was it intentional that this is using `:disabled` (not `[disabled]`)?

Reply:
oops, no good catch. I need to double check details/summary, I know one of these receives tab focus but I don't think both do.